src/js/components/chart/utils-normalization.js

```diff
@@ -0,0 +1,219 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import zip from 'lodash/zip';
```

Comment:
I removed the following lodash functions:
*flatten
*isInteger
*sortedIndexOf (although we are doing big data and it can get slow)

Regarding zip, not sure if there are other options.

All tests are passing after the refactoring :100: